src/main/java/com/worksap/nlp/sudachi/RegexOovProvider.java

```diff
@@ -149,4 +156,14 @@ private static Pattern checkPattern(String regex) {
         x.reset();
         return pattern;
     }
+
+    private static boolean isStrictContinuity(Settings settings) {
+        String content = settings.getString("boundaries", "strict");
```

Comment:
I think it would be easier to understand if the property name is strictBoundary and the value is true/false, but what do you think?

Reply:
I prefer enumerations in options as they are more extensible in the future. Here the possible extension is the mode which will ignore limitations on alphanumberic boundaries which are not hardcoded in the preprocessing.

Reply:
I agree.